cddl/cmw-collection.cddl

```diff
@@ -0,0 +1,7 @@
+cmw-collection = {
+  + cmw-collection-entry-label => cmw
+}
+
+cmw-collection-entry-label = text / int .feature "cbor"
+
+; cmw = cmw-array / cmw-cbor-tag<bytes> .feature "cbor"
```

Comment:
Is cmw-cbor-tag< > a macro? If so, where is it defined?

Reply:
> Is `cmw-cbor-tag< >` a macro? If so, where is it defined?

It's defined in [Figure 2](https://www.ietf.org/archive/id/draft-ietf-rats-msg-wrap-00.html#figure-2) of the CMW document.  It uses CDDL 2 ["Non-literal Tag Numbers"](https://www.ietf.org/archive/id/draft-ietf-cbor-update-8610-grammar-02.html#section-3.2).